man/src/image/tag.md

```diff
@@ -9,15 +9,16 @@ entire image name including the optional `TAG` after the ':'.
    present, it may optionally be followed by a port number in the format 
    `:8080`. If not present, the command uses Docker's public registry located at
    `registry-1.docker.io` by default. Name components may contain lowercase 
-   characters, digits and separators. A separator is defined as a period, one or 
+   letters, digits and separators. A separator is defined as a period, one or
    two underscores, or one or more dashes. A name component may not start or end 
```

Comment:
But hyphen is also used all over. I think that this is the best way of stating it: https://github.com/docker/docker/blame/master/man/docker-run.1.md#L614

All of this seems very UN-DRY but I have no idea how we'd fix that. Unless we want to start using a pre-processor on our Docs and use a "valid-characters" macro for all of the dozen places where we describe the same format :P